samples/snippets/nbactions.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Copyright & license on human created files

Reply:
Ew, I didn't mean to include that. Sorry! I'll remove that from this PR.